lightly/loss/barlow_twins_loss.py

```diff
@@ -77,8 +93,21 @@ def forward(self, z_a: torch.Tensor, z_b: torch.Tensor) -> torch.Tensor:
 def _normalize(
     z_a: torch.Tensor, z_b: torch.Tensor
 ) -> Tuple[torch.Tensor, torch.Tensor]:
-    """Helper function to normalize tensors along the batch dimension."""
+    """
+    Helper function to normalize tensors along the batch dimension.
```

Comment:
Just FYI, for private functions (starting with an underscore) it is not necessary to write very detailed docstrings. Exceptions are complex functions with a lot of logic.

Reply:
@guarin done.